src/steps/relay.py

```diff
@@ -34,6 +40,27 @@ def setup_main_relay_nodes(self, request):
         self.node2.start(relay="true", discv5_bootstrap_node=self.enr_uri)
         self.main_nodes.extend([self.node1, self.node2])
 
+    @pytest.fixture(scope="function")
+    def register_rln_main_relay_nodes(self, request):
```

Comment:
guess main_rln is better and alligned with the fixture bellow

Reply:
Fixed at https://github.com/waku-org/waku-interop-tests/pull/16/commits/b487d3f6609758677b277515a4b0eeb9f800d2f5